Bin/Documentation.php

```diff
@@ -309,7 +316,40 @@ public function main()
             }
         }
 
-        echo "\n", 'Open file://', $workspace, '/index.html', '.', "\n";
+        $pathname = escapeshellarg('file://' . $workspace . '/index.html');
+
+        echo "\n";
+
+        if (true === $open) {
+            if (isset($_SERVER['BROWSER'])) {
```

Comment:
It can but I don't see the point… multiple browsers in case of… what? I propose to go with this version. If someone encounters an issue, then we will fix it. It's more likely the `$BROWSER` will not be defined at all. Thoughts?